spring-integration-core/src/main/java/org/springframework/integration/core/Pausable.java

```diff
@@ -27,16 +28,18 @@
  * @since 5.0.3
  *
  */
-public interface Pausable extends Lifecycle {
+public interface Pausable extends ManageableLifecycle {
 
 	/**
 	 * Pause the endpoint.
 	 */
+	@ManagedOperation(description = "Pause the component")
```

Comment:
We export all components; they don't need `@ManagedResource`; they do, however, need `@IntegrationManagedResource` to prevent the normal MBE from exporting.

That said, pause/resume are not showing up - investigating...